### PR TITLE
feat: LikeCoin case study

### PR DESCRIPTION
**Pull request by jessicaschilling**
Add LikeCoin case study and corresponding links/nav furniture.
Preview: https://bafybeibaorbfivljbfstddjnqt6pvtty64hepdc2qnl7dmsca4htsmyklm.ipfs.dweb.link/concepts/case-study-likecoin/

**Please don't merge yet! Not marking as draft so as to generate a staging link for final review by internal/external stakeholders.**

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@autonome - over to you for another set of eyes. Once you've approved I'll send to the LikeCoin folks for final OK.

**Review by autonome — APPROVED** [review]
👍🏼👍🏼👍🏼

**Pull request merged**
